releng/release-notes-next/oraclelinux-distro_id.config

```diff
@@ -0,0 +1 @@
+Expand Oracle Linux distro_id from `ol` to `oraclelinux` when looking for configuration files [issue#1545]
```

Comment:
should be `[issue#1545][]`, but I can fix this later

Reply:
Oops, sorry. I completely missed that.